.travis.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           env: SYMFONY_VERSION="^5.0" MIN_STABILITY=dev
         - php: 7.4
           env: CHECKS=yes
+        - php: nightly
+          env: 'COMPOSER_FLAGS="--ignore-platform-reqs"'
```

Comment:
open question: what is failing without it ?

Reply:
phpunit/phpunit 9.3.9 requires phpspec/prophecy, but [it is not installable on PHP 8](https://travis-ci.org/github/php-coveralls/php-coveralls/jobs/730167751#L541).

Reply:
ack. thanks.